dirsrvtests/tests/suites/config/compact_test.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import pytest
+import os
+import time
+from lib389.tasks import DBCompactTask
+from lib389.backend import DatabaseConfig
+from lib389.topologies import topology_m1 as topo
+
+log = logging.getLogger(__name__)
+
```

Comment:
If you agree that this tests can be run as part of the PR CI, could you please add the following line here?

    pytestmark = pytest.mark.tier1